### ISSUE TITLE
The setting of hyperparameter

### ISSUE THREAD

**Issue by ZS123-lang** [baaivision/EVA#45]
Hi, Thanks for the great work.  I am confused about the meaning of  [pt_hw_seq_len](https://github.com/baaivision/EVA/blob/master/EVA-02/det/detectron2/modeling/backbone/vit.py#L313).  Is there any specific reason why it is set at 16? eg, following the pre-training setting, which is 224/14=16?  Can it be set as 1 or  being equal to  [ ft_seq_len](https://github.com/baaivision/EVA/blob/master/EVA-02/det/detectron2/modeling/backbone/vit.py#L373).  In such cases,  the position  [`t'](https://github.com/baaivision/EVA/blob/e06e475aa4df1e2ada1825564c0080776b2df78e/EVA-02/det/detectron2/modeling/backbone/utils.py#L334) is normalized to 0-1 or without normalization.

**Comment by Yuxin-CV** [comment]
Hi @ZS123-lang, we set `pt_hw_seq_len` to 16 because we want to align ft & pt, we also tried interpolate using `ft_seq_len` and we find it slightly degenerates the performance compared with using `pt_hw_seq_len` in IN-1K classification.

**Comment by Yuxin-CV** [comment]
Essentially, we tried two types of RoPE interpretation approaches:

1. Keep the total length same between ft & pt, and change the stride during ft if needed.
2. Keep the stride same between ft & pt, and change the total length during ft if needed.

We find these two approaches achieve similar performances in IN-1K classification (1 is our default choice)
